TelCo.ColorCoder/ColorPair.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    /// <summary>
+    /// The 25-pair color code, originally known as even-count color code, 
+    /// is a color code used to identify individual conductors in twisted-pair 
+    /// wiring for telecommunications.
+    /// This class provides the color coding and 
+    /// mapping of pair number to color and color to pair number.
+    /// </summary>
+    partial class Program
```

Comment:
Partial classes:
When a partial class is used, it comes with a cost of readability
Additional effort is required to decipher the partial class across multiple files.
Partial classes are mainly used 
- to splitting auto-generated code from main code.
- To de-clutter code for example for separating boiler plate code from main code
- In places where a class implements a lot of interfaces. Partial class can be used 1 file, 1 partial class for 1 interface implementation.

In cases other than the ones mentioned above, instead of partial class a new class can be created and aggregated to the main class.